### PR TITLE
chore: prep market creator for minting

### PR DESCRIPTION
**Pull request by DavidMinarsch**

### CODE REVIEW

File: mints/mapping.txt [hunk 1]
```diff
@@ -1,8 +1,15 @@
 token_id-"component_id"
-X-
+X-"contract/valory/conditional_tokens/0.1.0"
+X-"contract/valory/fpmm_deterministic_factory/0.1.0"
+X-"contract/valory/realtio/0.1.0"
+X-"contract/valory/wxdai/0.1.0"
+X-"contract/valory/fpmm/0.1.0"
+X-"skill/valory/market_creation_manager_abci/0.1.0" -> 3, 5, 14, 19, 20, 22, 57, 59, valory/realtio:0.1.0, valory/conditional_tokens:0.1.0, valory/fpmm_deterministic_factory:0.1.0, valory/wxdai:0.1.0, valory/fpmm:0.1.0
+X-"skill/valory/market_validation_abci/0.1.0" -> 14, 22
```
Comment (jmoreira-valory) [inline mints/mapping.txt:8]:
```suggestion
```

File: mints/mapping.txt [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+X-"skill/valory/market_maker_abci/0.1.0" -> 14, 16, 17, 22, 33, "skill/valory/market_creation_manager_abci/0.1.0", "skill/valory/market_validation_abci/0.1.0"
```
Comment (jmoreira-valory) [inline mints/mapping.txt:9]:
```suggestion
X-"skill/valory/market_maker_abci/0.1.0" -> 14, 16, 17, 22, 33, "skill/valory/market_creation_manager_abci/0.1.0"
```

File: mints/mapping.txt [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
 ---
 token_id-"agent_id"
-X-
+X-"agent/valory/market_maker/0.1.0" -> 1, 2, 3, 4, 5, 6, 7, 39, 57, 8, 9, 10, 11, 18, 19, 20, 12, 5 contracts above, 13, 14, 16, 17, 22, 33, 3 skills abovew
```
Comment (jmoreira-valory) [inline mints/mapping.txt:12]:
```suggestion
X-"agent/valory/market_maker/0.1.0" -> 1, 2, 3, 4, 5, 6, 7, 39, 57, 8, 9, 10, 11, 18, 19, 20, 12, 5 contracts above, 13, 14, 16, 17, 22, 33, 2 skills above
```

**Review by jmoreira-valory — CHANGES_REQUESTED** [review]
Please merge main as one component has been removed.

File: mints/02.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/conditional_tokens:0.1.0","description":"The scaffold contract scaffolds a contract to be implemented by the developer.","code_uri":"ipfs://bafybeihufvbfz7l7fb3fvsalp6jmplkxoc3rs3b25apultqmoelrvvx2de","image":"ipfs://bafybeif2uk24bpv55cxzjwwifqp7rxhzrveg2dp2ylg2x24pswy4tjv7iy","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```
Comment (DavidMinarsch) [inline mints/02.json:1]: "The scaffold contract scaffolds a contract to be implemented by the developer." -> you need to update the metadata of this package in its yaml @jmoreira-valory

File: mints/03.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/fpmm_deterministic_factory:0.1.0","description":"The scaffold contract scaffolds a contract to be implemented by the developer.","code_uri":"ipfs://bafybeibveawsp3ir67otkyswyfozetesjwrhyt5gfix6fanjwfq4dd3vyy","image":"ipfs://bafybeiayofa7yqccdzg5zkyyccyybj5wzaclpkyaom45oogmmg2rbdwwly","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```
Comment (DavidMinarsch) [inline mints/03.json:1]: "The scaffold contract scaffolds a contract to be implemented by the developer." -> you need to update the metadata of this package in its yaml @jmoreira-valory

File: mints/05.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/wxdai:0.1.0","description":"The scaffold contract scaffolds a contract to be implemented by the developer.","code_uri":"ipfs://bafybeicsbm2itxmmifkkyroozp6dh45aatrqabv6bcpkm7p2nin7j2trvm","image":"ipfs://bafybeicpuzx75bni7cjvhdnzp3id7ay5fzgmqvor37nxgjxiqwhwwjaayu","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```
Comment (DavidMinarsch) [inline mints/05.json:1]: "The scaffold contract scaffolds a contract to be implemented by the developer." -> you need to update the metadata of this package in its yaml @jmoreira-valory

File: mints/06.json [hunk 7]
```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/fpmm:0.1.0","description":"The scaffold contract scaffolds a contract to be implemented by the developer.","code_uri":"ipfs://bafybeihmzrpamqh654kb2vzzybzkirk75rmxj6rttrbhhkrju73jyjrjse","image":"ipfs://bafybeiabcpurda2ownkndcpchn5rtlwg6hk7jzweyb3q2y7wkxreo2l6uu","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```
Comment (DavidMinarsch) [inline mints/06.json:1]: "The scaffold contract scaffolds a contract to be implemented by the developer." -> you need to update the metadata of this package in its yaml @jmoreira-valory

File: mints/07.json [hunk 8]
```diff
@@ -0,0 +1 @@
+{"name":"skill/valory/market_creation_manager_abci:0.1.0","description":"The scaffold skill is a scaffold for your own skill implementation.","code_uri":"ipfs://bafybeihhtzwffrfs2owanv627tx53gtaapeelhwfemyzdmkysl7hwskf5q","image":"ipfs://bafybeifdvzkhdqhtit57ala2xorlf6tkl3p7di7bsl64yvhwvr575mon3y","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```
Comment (DavidMinarsch) [inline mints/07.json:1]: "The scaffold skill is a scaffold for your own skill implementation." -> you need to update the metadata of this package in its yaml @jmoreira-valory

File: mints/08.json [hunk 9]
```diff
@@ -0,0 +1 @@
+{"name":"skill/valory/market_maker_abci:0.1.0","description":"The scaffold skill is a scaffold for your own skill implementation.","code_uri":"ipfs://bafybeigw7ep2icp3xayuwkkuebw2vzahqfetryyxh5lekjjrdn4kyjkege","image":"ipfs://bafybeidqk5r4ju34cmmr7vsb7ewevc5xacfl3qdjvod556ds7fc3inzcpa","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```
Comment (DavidMinarsch) [inline mints/08.json:1]: See above

**Review by jmoreira-valory — APPROVED** [review]

**Pull request merged**